kubernetes/salt/config/master.d/salt-master.conf

```diff
@@ -5,11 +5,11 @@ interface: 0.0.0.0
 event_return: mysql
 
 mysql:
-  host: '127.0.0.1'
+  # salt does not support specifying the UNIX socket location here - as a workaround,
```

Comment:
Another option is to just make mysql listen on `127.0.0.1` (we would need to change less settings, and still it wouldn't be reachable from the outside).